File: week6/pset6/sentimental-cash/cash.py
from cs50 import get_float

# TODO
change = get_float("Change owed: ")

while True:
    if change < 0:
        change = get_float("Change owed: ")
    else:
        break

coins = 0

while round(change, 2) > 0.24:
    change = change - 0.25
    coins += 1

while round(change, 2) > 0.09:
    change = change - 0.1
    coins += 1

while round(change, 2) > 0.04:
    change = change - 0.05
    coins += 1

while round(change, 2) > 0:
    change = change - 0.01
    coins += 1

print(coins)

File: week1/pset1/cash_2021/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    float dollars;
    do
    {
        dollars = get_float("Change in dollars: ");
    }
    while (dollars < 0.00);

    int cents;
    {
        cents = round(dollars * 100);
    }

    int counter = 0;
    while (cents > 24)
    {
        cents = cents - 25;
        counter++;
    }
    while (cents > 9)
    {
        cents = cents - 10;
        counter++;
    }
    while (cents > 4)
    {
        cents = cents - 5;
        counter++;
    }
    while (cents > 0)
    {
        cents = cents - 1;
        counter++;
    }
    printf("%i\n", counter);
}